src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -187,6 +187,11 @@ export async function launchGame(
       return false
     }
 
+    appendGamePlayLog(
+      gameInfo,
+      `Launch Command: ${executable} ${launcherArgs ?? ''}\n`
```

Comment:
One issue I noticed with this is that it doesn't really print the launch command, it prints the executable and arguments only. So for a native game this is fine, but for a game using wine/proton this doesn't really show the actual command? for the different stores we print more data also like env variables for example

Reply:
I don't know how to convert the env variable record to a string, any guidance on how to do that is welcome

Reply:
```ts
Object.entries(envVars)
  .map(([name, value]) => `${name}=${value}`)
  .sort()
  .join(' ')
```
would be one way to do it